gnfish/alignment_trimming.py

```diff
@@ -6,35 +6,33 @@
 @author: hectorlorente
 """
 
-import re
-import argparse
+import click
 import subprocess
 
 
-def main():
-    # Arguments
-    parser = argparse.ArgumentParser(description="Alignment trimming using trimAl.")
-    parser.add_argument("trimal_path", help="path to trimAl directory", type=str)
-    parser.add_argument(
-        "input_file",
-        help="alignment (FASTA file with prot or nucl aligned seqs)",
-        type=str,
-    )
-    parser.add_argument("output_file", help="output file name.", nargs="?", type=str)
-    parser.add_argument(
-        "--trimal_parameters",
-        help="command to run trimal options. Default -gt 0.1",
-        nargs="?",
-        const="-gt 0.1",
-        type=str,
-        default="-gt 0.1",
-    )
-    args = parser.parse_args()
-    trimal_path = re.sub("'", "", args.trimal_path)
-    input_file = re.sub("'", "", args.input_file)
-    output_file = re.sub("'", "", args.output_file)
-    trimal_parameters = args.trimal_parameters
+@click.command()
+@click.argument("trimal_path", type=click.STRING)
+@click.argument(
+    "input_file",
+    type=click.STRING,
+)
+@click.argument("output_file", type=click.STRING)
+@click.option(
+    "--trimal_parameters",
+    help="Command to run trimal options. Default -gt 0.1",
+    type=click.STRING,
+    default="-gt 0.1",
+)
+def main(trimal_path, input_file, output_file, trimal_parameters):
+    """Trim an alignment using trimAl.
+
+    TRIMAL_PATH Path to trimAl directory.
```

Comment:
This path to software can maybe be simplified with conda env / containers. We will talk about this